addons/mail/static/src/new/messaging.js

```diff
@@ -7,6 +7,8 @@ import { sprintf } from "@web/core/utils/strings";
 import { url } from "@web/core/utils/urls";
 import { htmlToTextContentInline, convertBrToLineBreak, removeFromArray } from "./utils";
 import { prettifyMessageContent } from "./message_prettify_utils";
+import { Thread } from "./core/thread_model";
+import { Partner } from "./core/partner_model";
```

Comment:
I think it will quickly be very convenient if we could import all the models from the same file.